<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JointMotionScope</class>
 <widget class="QWidget" name="JointMotionScope">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1700</width>
    <height>944</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>1190</x>
     <y>30</y>
     <width>480</width>
     <height>774</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#groupBox{background: #FFFFFF;
border-radius: 12px;}</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>164</x>
      <y>40</y>
      <width>151</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>测试</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="allOut_Btn">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>122</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{border-image: url(:/new/prefix1/source/jointScope/allOutMormal.png);}
QPushButton:hover{border-image: url(:/new/prefix1/source/jointScope/allOutHover.png);}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="fourInOneOut_Btn">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>122</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{border-image: url(:/new/prefix1/source/jointScope/fourInOneOurNormal.png);}
QPushButton:hover{border-image: url(:/new/prefix1/source/jointScope/fourInOneOUtHover.png);}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="fourOutOneIn_Btn">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>122</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{border-image: url(:/new/prefix1/source/jointScope/fourOutOneInNormal.png);}
QPushButton:hover{border-image: url(:/new/prefix1/source/jointScope/fourOutOneInHover.png);}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="lock_Btn">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>256</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{border-image: url(:/new/prefix1/source/jointScope/unlockNormal.png);}
QPushButton:hover{border-image: url(:/new/prefix1/source/jointScope/unlockHover.png);}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="reset_Btn">
   <property name="geometry">
    <rect>
     <x>1210</x>
     <y>850</y>
     <width>164</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{background: #0D9DDB;border-radius: 8px;color:white;}
QPushButton:hover{background: #1EAACF;border-radius: 8px;color:#333333;}</string>
   </property>
   <property name="text">
    <string>开始测试</string>
   </property>
  </widget>
  <widget class="QPushButton" name="save_Btn">
   <property name="geometry">
    <rect>
     <x>1480</x>
     <y>850</y>
     <width>164</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{background: #0D9DDB;border-radius: 4px;color:white;}
QPushButton:hover{background: #1EAACF;border-radius: 4px;color:#333333;}</string>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>30</y>
     <width>640</width>
     <height>720</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="leftHand_Page">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>640</width>
       <height>720</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox{border:none;}</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QPushButton" name="reset4L_Btn">
      <property name="geometry">
       <rect>
        <x>108</x>
        <y>126</y>
        <width>36</width>
        <height>36</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{border-image: url(:/new/prefix1/source/image/resetNormal.png);}
QPushButton:hover{border-image: url(:/new/prefix1/source/image/resetHover.png);}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="reset3L_Btn">
      <property name="geometry">
       <rect>
        <x>178</x>
        <y>71</y>
        <width>36</width>
        <height>36</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{border-image: url(:/new/prefix1/source/image/resetNormal.png);}
QPushButton:hover{border-image: url(:/new/prefix1/source/image/resetHover.png);}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="reset2L_Btn">
      <property name="geometry">
       <rect>
        <x>247</x>
        <y>27</y>
        <width>36</width>
        <height>36</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{border-image: url(:/new/prefix1/source/image/resetNormal.png);}
QPushButton:hover{border-image: url(:/new/prefix1/source/image/resetHover.png);}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="reset1L_Btn">
      <property name="geometry">
       <rect>
        <x>321</x>
        <y>48</y>
        <width>36</width>
        <height>36</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{border-image: url(:/new/prefix1/source/image/resetNormal.png);}
QPushButton:hover{border-image: url(:/new/prefix1/source/image/resetHover.png);}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="reset0L_Btn">
      <property name="geometry">
       <rect>
        <x>572</x>
        <y>455</y>
        <width>36</width>
        <height>36</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{border-image: url(:/new/prefix1/source/image/resetNormal.png);}
QPushButton:hover{border-image: url(:/new/prefix1/source/image/resetHover.png);}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="CHandScopeWidget" name="leftHand_Widget" native="true">
      <property name="geometry">
       <rect>
        <x>105</x>
        <y>88</y>
        <width>434</width>
        <height>563</height>
       </rect>
      </property>
      <widget class="QLabel" name="LTouch_Label">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>220</y>
         <width>81</width>
         <height>101</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/new/prefix1/source/dialog/touchWarning.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <zorder>leftHand_Widget</zorder>
     <zorder>reset4L_Btn</zorder>
     <zorder>reset3L_Btn</zorder>
     <zorder>reset2L_Btn</zorder>
     <zorder>reset1L_Btn</zorder>
     <zorder>reset0L_Btn</zorder>
    </widget>
   </widget>
   <widget class="QWidget" name="rightHand_Page">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>640</width>
       <height>720</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox{border:none;}</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QPushButton" name="reset1R_Btn">
      <property name="geometry">
       <rect>
        <x>356</x>
        <y>48</y>
        <width>36</width>
        <height>36</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{border-image: url(:/new/prefix1/source/image/resetNormal.png);}
QPushButton:hover{border-image: url(:/new/prefix1/source/image/resetHover.png);}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="reset2R_Btn">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>27</y>
        <width>36</width>
        <height>36</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{border-image: url(:/new/prefix1/source/image/resetNormal.png);}
QPushButton:hover{border-image: url(:/new/prefix1/source/image/resetHover.png);}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="reset3R_Btn">
      <property name="geometry">
       <rect>
        <x>499</x>
        <y>71</y>
        <width>36</width>
        <height>36</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{border-image: url(:/new/prefix1/source/image/resetNormal.png);}
QPushButton:hover{border-image: url(:/new/prefix1/source/image/resetHover.png);}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="reset4R_Btn">
      <property name="geometry">
       <rect>
        <x>569</x>
        <y>126</y>
        <width>36</width>
        <height>36</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{border-image: url(:/new/prefix1/source/image/resetNormal.png);}
QPushButton:hover{border-image: url(:/new/prefix1/source/image/resetHover.png);}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="reset0R_Btn">
      <property name="geometry">
       <rect>
        <x>105</x>
        <y>455</y>
        <width>36</width>
        <height>36</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{border-image: url(:/new/prefix1/source/image/resetNormal.png);}
QPushButton:hover{border-image: url(:/new/prefix1/source/image/resetHover.png);}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="CHandScopeWidget" name="rightHand_Widget" native="true">
      <property name="geometry">
       <rect>
        <x>174</x>
        <y>88</y>
        <width>434</width>
        <height>563</height>
       </rect>
      </property>
      <widget class="QLabel" name="RTouch_Label">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>210</y>
         <width>81</width>
         <height>101</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/new/prefix1/source/dialog/touchWarning.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <zorder>rightHand_Widget</zorder>
     <zorder>reset1R_Btn</zorder>
     <zorder>reset2R_Btn</zorder>
     <zorder>reset3R_Btn</zorder>
     <zorder>reset4R_Btn</zorder>
     <zorder>reset0R_Btn</zorder>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CHandScopeWidget</class>
   <extends>QWidget</extends>
   <header>chandscopewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
